tests/playwright-test/test-output-dir.spec.ts

```diff
@@ -429,6 +429,21 @@ test('should allow nonAscii characters in the output dir', async ({ runInlineTes
   expect(outputDir).toBe(path.join(testInfo.outputDir, 'test-results', 'my-test-こんにちは世界'));
 });
 
+test.only('should allow shorten long output dirs characters in the output dir', async ({ runInlineTest }, testInfo) => {
+  const result = await runInlineTest({
+    'my-test.spec.js': `
+      const { test } = pwt;
+      test.describe('this is a really long description that would be too long for a file path', () => {
+        test('and this is an even longer test name that just keeps going and going and we should shorten it', async ({}, testInfo) => {
+          console.log('\\n%%' + testInfo.outputDir);
+        });
+      });
+    `,
+  });
+  const outputDir = result.output.split('\n').filter(x => x.startsWith('%%'))[0].slice('%%'.length);
+  expect(outputDir).toBe(path.join(testInfo.outputDir, 'test-results', 'my-test-this-is-a-really-long-descripti-f9015f78f2853e47676ac0890dcea2f2b85-d-going-and-we-should-shorten-it'));
```

Comment:
this still contains the full hash, shouldn't it be only the last 5 chars of the sha1?

Reply:
Yep. Silly mistake.